firmware/klipper_configurations/SKR_1.4/Voron2_SKR_14_Config.cfg

```diff
@@ -162,7 +162,7 @@ homing_positive_dir: true
 ##	Make sure to update below for your relevant driver (2208 or 2209)
 [tmc2209 stepper_y]
 uart_pin: P1.9
-interpolate: True
+interpolate: false
 run_current: 0.8
 hold_current: 0.7
```

Comment:
missed change on hold_current. Should be 0.8

Reply:
Thanks for the catch. Fixed.

Reply:
Why should I turn off interpolate? I can't see the benefits of turning off interpolate. Enabling interpolation can often be quieter and achieve more accurate step movement.

Reply:
@jack9603301 Please take a look at this pull request. It goes deeply in to details. https://github.com/Klipper3d/klipper/pull/4977

Reply:
doesn't that recommend 64 microsteps, or even 128 due to noise? 32 seem low.

Reply:
I haven't done any testing of microstep values greater than 64, and I haven't tested 64 microsteps on a Pi zero to verify it doesn't overload the system. That's why I chose 32 bit values to just play it safe.

Reply:
I have done so far ~40 hours of printing since the change. 32 was on the bit louder side and those noises were really odd. So I went from 32 to 64 microsteps and the latter is stable and quiet(close to what it was with interpolate:true if not even better). Zero handled interpolation just fine, so it might be no issue. Though if someone could verify the load would be great.